houses/views.py

```diff
@@ -1,3 +1,68 @@
-from django.shortcuts import render
+import json
+
+from django.views            import View
+from django.http             import JsonResponse
+
+from .models                 import House, HouseImage, HouseType, Ghost ,Country,City
+from reservations.models     import Reservation
+
+
+
+class HouseDetailView(View):
+    def get(self, request, house_id):
+        houses = House.objects.filter(id = house_id)
+        houseimage = HouseImage.objects.filter(id=house_id)
+        house = House.objects.get(id=house_id)
+        result = [
+        {
+            'house_id': house.id,
+            'name' : house.name,
+            'description' : house.description,
+            'latitude' : house.latitude,
+            'longitude': house.longitude,
+            'house_type': house.house_type.name,
+            'max_guest' : house.max_guest,
+            'country': house.city.country.name,
+            'city' : house.city.name,
+            'reservation': house.reservation_set.all(),
+            'trap' : house.trap,
+            'exit' : house.exit,
+            'houseimage' : [
+                    houseimage.image_url.split('/')[-1].split('.')[0]
+                    for houseimage in house.houseimage_set.all()],
+        }for house in houses]
```

Comment:
- 피드백 3. 해당 View는 `HouseDetailView`로 House 정보 하나만 반환하는 View가 되어야 합니다. 그런데 결과(result)는 리스트로 반환하고 있습니다. 아래와 같이 house 객체 하나만 반환하도록 해주세요.

```suggestion
        result = 
        {
            'house_id': house.id,
            'name' : house.name,
            'description' : house.description,
            'latitude' : house.latitude,
            'longitude': house.longitude,
            'house_type': house.house_type.name,
            'max_guest' : house.max_guest,
            'country': house.city.country.name,
            'city' : house.city.name,
            'reservation': house.reservation_set.all(),
            'trap' : house.trap,
            'exit' : house.exit,
            'houseimage' : [
                    houseimage.image_url.split('/')[-1].split('.')[0]
                    for houseimage in house.houseimage_set.all()],
        }
```